wdae/wdae/gene_sets/views.py

```diff
@@ -226,3 +232,15 @@ def get(self, _request: Request) -> Response:
         if self.gpf_instance.denovo_gene_sets_db.has_gene_sets():
             return Response(status=status.HTTP_204_NO_CONTENT)
         return Response(status=status.HTTP_404_NOT_FOUND)
+
+
+def gene_sets_types_to_dict(gene_sets_types: list[Any]) -> dict[str, Any]:
+    """Convert typescript style gene sets types into python dict."""
+    gene_sets_types_dict = {}
```

Comment:
this should be renamed to something more appropriate
tracing where gene_sets_types is used leads to DenovoGeneSetsDb.get_all_gene_sets where it is referred to as `denovo_gene_set_spec`, which is still not very good but not as abstract